cpg_workflows/jobs/picard.py

```diff
@@ -144,12 +153,16 @@ def markdup(
     assert isinstance(j.output_cram, hb.ResourceGroup)
     cmd = f"""
     picard MarkDuplicates -Xms{resource.get_java_mem_mb()}M \\
-    I={sorted_bam} O=/dev/stdout M={j.markdup_metrics} \\
+    I={sorted_bam} O={j.output_bam} M={j.markdup_metrics} \\
```

Comment:
Possibly call this `j.temp_bam` or so, so that which is in and which is out in the `samtools view` arguments below is clearer. But no biggie.